src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -235,7 +237,16 @@ def run(self) -> List[nodes.Node]:  # noqa: WPS210
                 config_path=config.towncrier_draft_config_path,
             )
         except subprocess.CalledProcessError as proc_exc:
```

Comment:
I still maintain that the underlying subprocess exception shouldn't leak here but instead should be turned into a `RuntimeError` where it happened (processed here). I've explained this @ https://github.com/sphinx-contrib/sphinxcontrib-towncrier/pull/58#discussion_r952615118.